x-pack/plugins/security_solution/public/management/components/policy_response/policy_response.tsx

```diff
@@ -155,37 +173,100 @@ export const PolicyResponse = memo(
       ]
     );
 
-    const getResponseConfigs = useCallback(
-      () =>
-        Object.entries(policyResponseConfig).map(([key, val]) => {
-          const attentionCount = policyResponseAttentionCount.get(key);
+    const getArtifacts = useCallback(
+      (
+        artifactIdentifiers: PolicyResponseProps['policyResponseArtifacts'][
+          | 'global'
+          | 'user']['identifiers']
+      ) => {
+        return artifactIdentifiers.map((artifact) => {
```

Comment:
I see multiple nested bjects here. Are we sure `artifactIdentifiers` always exist?